debian/postinst

```diff
@@ -18,7 +18,13 @@ case "$1" in
     # create nginx configurations:
     ln -sf /etc/kolibri/dist/nginx.conf /etc/nginx/conf.d/kolibri.conf
     db_get kolibri-server/port
-    su $KOLIBRI_USER -c "/usr/share/kolibri-server/kolibri_server_setup.py -d $RET"
+    PORT=$RET
+    su $KOLIBRI_USER -c "/usr/share/kolibri-server/kolibri_server_setup.py -d $PORT"
+    if [ -L "/etc/nginx/sites-enabled/default" ] && [ "$PORT" = "80" ] ;then
+        rm  /etc/nginx/sites-enabled/default
+    elif [ ! -L "/etc/nginx/sites-enabled/default" ] && [ "$PORT" != "80" ] ;then
+        ln -s /etc/nginx/sites-available/default /etc/nginx/sites-enabled/default
+    fi
```

Comment:
The latter part (re-enabling the symlink) would be annoying to some things that I work on because I remove `default` and activate something else in its place.

Reply:
Yes, I have thought of it, but I want to re-enable it if we disable it. I'll think of some flag file to do it because I don't want users to lose the default behaviour if they remove the package.

Reply:
That sounds like the right way to do it! Shouldn't it also have an additional restore in postrm?

Reply:
@benjaoming  Yep, done, please, take a look.